### ISSUE TITLE
Python `SyntaxError` when removing last import from parenthetical import ... from statement

### ISSUE THREAD

**Issue by Jasha10** [lyz-code/autoimport#178]
## Description
When autoimport removes the last import from a multi-line parenthetical import statement, the result gives a `SyntaxError`.

## Steps to reproduce
1. Start with the following file:
```python
from foo import (
    bar,
    baz,
)
```
2. Run `autoimport`
3. The file now looks like this:
```python
from foo import (
)
```
4. Run python, get a `SyntaxError`.

## Desired behavior
Autoimport should produce valid python code. The `from foo import (\n)` should be removed.

## Environment
```text
>>> print(autoimport.version.version_info())
           autoimport version: 1.0.0
               python version: 3.9.9 | packaged by conda-forge | (main, Dec 20 2021, 02:40:17)  [GCC 9.4.0]
                     platform: Linux-5.4.0-91-generic-x86_64-with-glibc2.27
```

**Comment by lyz-code** [comment]
Hi @Jasha10 thanks again for opening the issue, I'll look into it next week, as always, if you can make a PR I'll be grateful

**Comment by lyz-code** [comment]
It should be fixed by https://github.com/lyz-code/autoimport/pull/180 and available from `1.0.1`, please check if it works for you

**Comment by Jasha10** [comment]
Yes, it's working, thanks!
